### DISCUSSION TITLE
How to get started?

### DISCUSSION THREAD

**Discussion by DGrothe-PhD** [ashishpawar517/AutoCompiler#21] [Q&A] [answered]
Hi,

after cloning and checkout of new_update branch, I tried to compile this project. Usually I create an Eclipse project and then put my *.java files all in the same folder and simply do "Run" within Eclipse IDE, that works fine for me. So this folder structure is somewhat different and I do not how this type of project here might be compiled, and this is what I've been trying to do so far:

So from within `...\AutoCompiler\src\main\java\com\ap\autocompiler\main`,
I typed `java Main.java`, got plenty of errors, then: 
`javac -d Main.java`, answer: <em>error: no source files</em>.

Any suggestions?

**Comment by ashishpawar517** [comment, marked as answer]
I use the maven plugin for my development purpose on VSCode if you have IntelliJ just click a run button on top of the main class. this should resolve your problem, Also we'll connect separately on this to resolve your issue.
